### PR TITLE
Homework12: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

File: homework12_card.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        """Fills the deck with regular cards and two jokers."""
+        suits = ["Hearts", "Diamonds", "Clubs", "Spades"]
+        numbers = list(range(1, 14))
+
+        for suit in suits:
+            for number in numbers:
+                self.cards.append(Card(number, suit))
+
+        self.cards.append(Card(14, "Joker"))
+        self.cards.append(Card(15, "Joker"))
```
Comment (Stml89) [inline homework12_card.py]:
```suggestion
        self.cards.append(Card("Red", "Joker"))
        self.cards.append(Card("Black", "Joker"))
```

File: homework12_currency.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        """Function takes the deposit amount and term and returns the total
+         amount in the user's account."""
+        deposit = Deposit(sum_of_deposit, deposit_term)
+        final_amount1 = deposit.final_amount()
+        self.user_accounts[user1] = final_amount1
+        return final_amount1
+
+    def exchange_currency(self, amount, from_currency, to_currency='BYN'):
+        """Function converts money from one currency to another."""
+        if from_currency == "BYN":
```
Comment (Stml89) [inline homework12_currency.py]: what if `to_currency` would be `USD`? You are gonna to skip this case

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework12_card.py (hunk 1 above)
Comment (AlinaShautsova) [reply homework12_card.py]: corrected

File: homework12_currency.py (hunk 2 above)
Comment (AlinaShautsova) [reply homework12_currency.py]: fixed it

**Review by Stml89 — APPROVED** [review]

**Pull request merged**